samples/c2pa/monolithic/c2pa-demo-monolithic.html

```diff
@@ -0,0 +1,101 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <title>C2PA Validation for Monolithic MP4</title>
+
+    <!-- Bootstrap core CSS -->
+    <link href="../../lib/bootstrap/bootstrap.min.css" rel="stylesheet">
+    <link href="../../lib/main.css" rel="stylesheet">
+
+    <!-- video-js -->
+    <script src="https://vjs.zencdn.net/8.3.0/video.min.js"></script>
+    <link href="https://vjs.zencdn.net/8.3.0/video-js.css" rel="stylesheet" />
+
+    <!-- c2pa player -->
+    <link href="../c2pa-player.css" rel="stylesheet" />
+    <script class="code" src="../c2pa-player.js"></script>
+
+    <style>
+        video {
+            width: 640px;
+            height: 360px;
+        }
+    </style>
+
+    <script type="module" class="code">
+        /* Import c2pa plugin */
+        import {c2pa_init} from './c2pa-monolithic-plugin.js';
+
+        /* c2pa player instance */
+        var c2paPlayer;
+        
+        function init() {
+            var video,
+                url = "../../microsoft/video_samples/monolithic/provenanceoutput_progressivedownload.mp4";
```

Comment:
I tested using this one `../../microsoft/video_samples/provenanceoutput_tearsOfSteel_flatfile_progressivedownload.mp4`

Reply:
Yes, that's what I used as well